ui/app/routes/vault/cluster/dashboard.js

```diff
@@ -9,10 +9,32 @@ import { hash } from 'rsvp';
 
 export default class VaultClusterDashboardRoute extends Route {
   @service store;
+  @service version;
+
+  async getVaultConfiguration() {
+    try {
+      const adapter = this.store.adapterFor('application');
+      const configState = await adapter.ajax('/v1/sys/config/state/sanitized', 'GET');
+      return configState.data;
+    } catch (e) {
+      return e.httpStatus;
+    }
+  }
 
   model() {
     return hash({
       secretsEngines: this.store.query('secret-engine', {}),
+      version: this.version,
+      vaultConfiguration: this.getVaultConfiguration(),
     });
   }
+
+  setupController(controller, resolvedModel) {
+    super.setupController(controller, resolvedModel);
+    const { version } = resolvedModel;
```

Comment:
version could be accessed by `this.version` so I don't think we need this extra variable declaration here